platforms/core-configuration/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/accessors/ProjectAccessorsClassPathTest.kt

```diff
@@ -172,14 +172,25 @@ class ProjectAccessorsClassPathTest : AbstractDslTest() {
                     dependencyHandlerExtensionMethods(name).forEach {
                         assertEquals(
                             isDeprecated(it),
-                            config.hasDeclarationDeprecations()
+                            config.hasDeclarationDeprecations() || isDeprecatedAccessor(it)
                         )
                     }
                 }
             }
         }
     }
 
+    /**
+     * Determines whether the method is the multi-string accessor, which is deprecated
+     * for removal in Gradle 10.
+     */
+    private
+    fun isDeprecatedAccessor(method: Method): Boolean =
+        method.parameters.map { it.type } == listOf(
```

Comment:
❓ Does this need to check for other numbers of string args here?  This seems like the max possible # of Strings vs. just using the G,A, and V and nothing else.

Reply:
This code is testing the generated accessors for configurations. 

Gradle generates these accessors for deprecated configurations, but marks them deprecated. 

This test verifies that _only_ generated accessors for deprecated configurations are marked deprecated. 

However, after these changes, we've _also_ deprecated one method in the generated accessors that accepts this exact set of parameters. 

Now, we want the test to verify that if a method is found to be deprecated, it is _either_ this specific method, or it is from a deprecated configuration. 

In short, no we don't need to check other parameter lists